### ISSUE TITLE
Recently closed tabs

### ISSUE THREAD

**Issue by anewuser** [Bill13579/tabby#50]
Have you considered adding a list of recently closed tabs, so that users can reopen them? Check this out: https://github.com/reblws/tab-search

**Comment by GabrielGab** [comment]
Agree and add: or some mechanism to drag from Firefox Recent History of website visits

**Comment by rushowr** [comment]
Opening the menu -> history -> recently closed tabs allows you to have this functionality. Then, if you open the tab you want, you can then manage it with Tabby. Not saying it's a fix, just that the functionality specifically requested exists elsewhere.

**Comment by rayman89** [comment]
Yes if we can get a button tab or search command to show the list to select which ones to restore would be great
